api/controllers/ABModelController.js

```diff
@@ -1426,6 +1477,7 @@ module.exports = {
                     .catch(res.AD.error)
```

Comment:
Should we resolvePendingTransaction() here?
```suggestion
                    .catch(err => {
                        resolvePendingTransaction();
                        res.AD.error(err);
                    })
```

Reply:
Ok, good .catch().